sep.pyx

```diff
@@ -873,8 +902,13 @@ def sum_circle(np.ndarray data not None, x, y, r,
     cdef int status
     cdef np.broadcast it
     cdef sep_image im
-
-    _parse_arrays(data, err, var, mask, &im)
+            
+    # Test for seg without seg_id.  Nothing happens if seg_id supplied but 
+    # without seg.
+    if (seg is not None) & (seg_id is None):
```

Comment:
This is very minor, but probably better to use `and` instead of `&` here.

Reply:
Fair enough (f25b42d7f10323b1d598e6aa65037a7c8dbd14bb).  I never use `and` but perhaps I should....

Reply:
Yeah, definitely not a big deal. Generally the benefit of `and` is that it is short-circuiting. Here that is irrelevant (`seg_id is None` takes very little time anyway), so the only benefit is clarity of intent: so the reader isn't asking "wait, why is a bitwise operation necessary here?"